res/lang/rooms/rooms_de.txt

```diff
@@ -9,10 +9,10 @@ ROOM_MOVIEAGENCY         = Film-Agentur
 ROOM_MOVIEAGENCY_OWNER   = J. Steward Jabble
 ROOM_ADAGENCY            = Agentur für Werbung
```

Comment:
Dann waere auch hier das "für" zu viel und es sollte "Werbeagentur" heissen.

Allerdings gibt es ja mit Absicht manchmal so eigenwillige Eigennamen: "[Die] Agentur für Werbung"

Reply:
Ja die Bezeichnungen sind auch nicht ganz einheitlich "Film-Agentur" vs. "Drehbuchagentur". Bei den eigenen Räumen habe ich es einheitlich gemacht, weil das Nachrichtenstudio so lang ist, dass beim eigenen Studio schon der Sender abgeschnitten ist.

Vereinheitlichungsvorschlag in separatem Commit